docs/user/docs/getting-started/working-with-git.md

```diff
@@ -110,13 +110,136 @@ there. Having done that you can click "Push".
 Now your changes are saved at the remote repository where they are not lost and
 your collegues can see them.
 
+## Diff and Merge Models
+
+<!-- prettier-ignore -->
+!!! warning
+   If you have to merge due to a merge conflict do not commit unless all merges highlighted are solved.
+
+If the merge tool is not opening on its own due to a merge conflict you can
+also diff two models on your own in order to check which changes were made.
+
+Therefore, select both `.aird`-files (e.g. with control click). Then right
+click on one of the files and select "Compare with" > "Each other as model".
+
+![Open compare View](working-with-git/start-compare-view.png)
+
+In the image you can see the
+[coffee machine model](https://github.com/DSD-DBS/coffee-machine) imported
+twice with different changes made to each model in order to compare them.
+
+In the View "Define the comparison operation to perform" it is important to
+choose the **correct** order of which model is the and which one is the target.
+In that case we are taking the changes in the source to apply them to the
+target and are going to change the target model.
+
+<!-- prettier-ignore -->
+!!! warning
+
+Make sure to select the correct direction of merging in order to modify the
+correct model.
+
+![Configure Merge Direction](working-with-git/configure-merge-direction.png)
+
+In the example we want to merge our changes from Left (coffee-machine model on
+working branch) into Right (coffee machine model on main branch)(1).
+
+Afterwards click "Configure" (2) to make further configurations.
+
+Choose the option "Transfer of elements between models created independently".
+Then click "Show advanced settings >>" to configure the matching process
+between elements. Thus, click the now appeared tab "Matching".
+
+Usually the UUIDs of the objects in your model should be stable. So ticking the
+Absolute criteria "Technical IDs" and "Property IDs" should be sufficient.
+Especially for if you know that there could be changes in diagrams you might
+also want to tick "Diagrams: Match shapes according to represented elements"
+and "Diagrams: Match remaining shapes according to type" as in Diagrams IDs
+might change just by opening them. If you are unsure whether your IDs are
+stable the option "Capella: Match equivalent technical elements" also worked in
+the past. If nothing else helps choose "Names" and click "Use labels as names"
```

Comment:
The sentense "If you ... worked in the past" is a bit weird. Maybe it's something for an info block. It may work for a user, but he has to experiment.

Reply:
Do you think that is an option which we should leave out in that case?

Reply:
I'd remove the sentence completely.